client/src/components/common/applicant-info-3.vue

```diff
@@ -74,7 +74,7 @@
         <v-col cols="2" class="h6">About The Business</v-col>
         <v-col cols="5" align-self="start">
           <v-tooltip top
-            content-class="top-tooltip contact-tooltip"
```

Comment:
Can "contact-tooltip" be deleted from the local styles now?

Reply:
Yes. I missed it